util/cliutil/util.go

```diff
@@ -211,5 +218,15 @@ func SetupDatabase(dburl string, maxConnections int) (*gorm.DB, error) {
 	sqldb.SetMaxOpenConns(openConns)
 	sqldb.SetConnMaxIdleTime(time.Hour)
 
+	if isSqlite {
+		// Set pragmas for sqlite
+		if err := db.Exec("PRAGMA journal_mode=WAL;").Error; err != nil {
+			return nil, err
+		}
+		if err := db.Exec("PRAGMA synchronous=normal;").Error; err != nil {
+			return nil, err
+		}
+	}
```

Comment:
is this going to impact any other prod services? does Relay use sqlite?

Reply:
Other things use sqlite (not relay) but they all want these pragmas anyhow, this is our standard practice for prod sqlite.